Calculator/CalculatorItemQueueTests/CalculatorItemQueueTests.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  CalculatorItemQueueTests.swift
+//  CalculatorItemQueueTests
+//
+//  Created by jiye Yi on 2023/01/25.
+//
+
+import XCTest
+@testable import Calculator
+final class CalculatorItemQueueTests: XCTestCase {
+    var sut: CalculatorItemQueue<String>!
+
+    override func setUpWithError() throws {
+      try super.setUpWithError()
+        sut = CalculatorItemQueue()
+    }
+
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        sut = nil
+    }
+    func test_배열이비어있을때_true를반환한다() {
+        sut.input = []
+        sut.output = []
+        let expectation = true
+        
+        let result = sut.isEmpty
+        
+        XCTAssertEqual(result, expectation)
+    }
+    func test_input_output_배열의수를반환한다() {
```

Comment:
```suggestion
    }

    func test_배열이비어있을때_true를반환한다() {
        sut.input = []
        sut.output = []
        let expectation = true
        
        let result = sut.isEmpty
        
        XCTAssertEqual(result, expectation)
    }
    
    func test_input_output_배열의수를반환한다() {
```
메서드 앞뒤에 개행이 빠진 것 같아요!

Reply:
넵 수정하였습니다!